drivers/can/Kconfig.native_posix

```diff
@@ -0,0 +1,28 @@
+# Kconfig.native_posix - native_posix CAN configuration options
+
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_NATIVE_POSIX
+	bool "native_posix CAN Driver"
+	depends on ARCH_POSIX
+	help
+	  Enable native_posix CAN driver
+
+config CAN_MAX_FILTER
+	int "Maximum number of concurrent active filters"
+	depends on CAN_NATIVE_POSIX
+	default 32
+	range 1 56
+	help
+	  Defines the array size of the callback/msgq pointers.
+	  Must be at least the size of concurrent reads.
+
+config CAN_NATIVE_POSIX_INTERFACE_NAME
```

Comment:
Sidecomment: If you want you can add a command line option to set it, similarly to:
https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/bluetooth/hci/userchan.c#L239

Reply:
Thanks for the hint. That looks like a way to go.